dataland-frontend/tests/e2e/fixtures/p2p/P2pPreparedFixtures.ts

```diff
@@ -0,0 +1,84 @@
+import { FixtureData } from "@sharedUtils/Fixtures";
+import { PathwaysToParisData } from "@clients/backend";
+import { generateP2pFixture } from "./P2pDataFixtures";
+
+/**
+ * Generates LkSG prepared fixtures by generating random LkSG datasets and afterwards manipulating some fields
+ * via manipulator-functions to set specific values for those fields.
+ * @param undef_probability probability for a field to be undefined
+ * @param toggleRandomSectors determines if the sector list should include all possible sectors or a randomized selection
+ * @returns the prepared fixtures
+ */
+export function generateP2pPreparedFixtures(
+  undef_probability = 0.5,
+  toggleRandomSectors = true
+): Array<FixtureData<PathwaysToParisData>> {
+  const preparedFixtures = [];
+  preparedFixtures.push(
+    manipulateFixtureForSixP2pDataSetsInDifferentYears(generateP2pFixture(1, undef_probability, toggleRandomSectors)[0])
```

Comment:
generateP2pFixture potentially multiple fixtures. So let's rename it to generateP2pFixtures

Reply:
Changed